skimage/measure/simple_metrics.py

```diff
@@ -142,3 +144,47 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def compare_nmi(im_true, im_test, *, bins=100):
+    """Compute the normalized mutual information.
+
+    The normalized mutual information is given by:
+
+                H(A) + H(B)
+      Y(A, B) = -----------
+                  H(A, B)
+
+    where H(X) is the entropy ``- sum(x log x) for x in X``.
+
+    It was proposed to be useful in registering images by Colin Studholme and
+    colleagues [1]_.
+
+    Parameters
+    ----------
+    im_true, im_test : ndarray
+        Images to be compared.
+    bins : int or sequence of int, optional
+        The number of bins along each axis of the joint histogram.
+
+    Returns
+    -------
+    nmi : float
+        The normalized mutual information between the two arrays, computed at
```

Comment:
should be said if lower NMI is better or not...
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.normalized_mutual_info_score.html